src/Sulu/Bundle/ContactBundle/Tests/Unit/CustomerIdConverterTest.php

```diff
@@ -32,7 +32,7 @@ public function convertIdsToGroupedIdsProvider()
     /**
      * @dataProvider convertIdsToGroupedIdsProvider
      */
-    public function testConvertIdsToGroupedIds($ids, $default, $expected): void
+    public function testConvertIdsToGroupedIds(array $ids, array $default, $expected): void
```

Comment:
This is not a bc break because the function where those variables are used already have a return type of array. So the only thing is we fail the code faster on broken types.